starport/services/serve/serve.go

```diff
@@ -450,6 +672,22 @@ func (s *Serve) serverSteps(conf starportconf.Config) (steps step.Steps) {
 		fmt.Fprintf(s.stdLog(logStarport).out, "🌍 Running a Cosmos '%[1]v' app with Tendermint at %s.\n", s.app.Name, xurl.HTTP(conf.Servers.RPCAddr))
 		fmt.Fprintf(s.stdLog(logStarport).out, "🌍 Running a server at %s (LCD)\n", xurl.HTTP(conf.Servers.APIAddr))
 		fmt.Fprintf(s.stdLog(logStarport).out, "\n🚀 Get started: %s\n\n", xurl.HTTP(conf.Servers.DevUIAddr))
+		time.Sleep(time.Second * 10)
```

Comment:
Why do we wait 10 seconds?

Reply:
this is removed in https://github.com/tendermint/starport/pull/336